tests/cnf/core/network/sriov/tests/sriov-bond-cni.go

```
@@ -0,0 +1,907 @@
+package tests
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/rh-ecosystem-edge/eco-goinfra/pkg/namespace"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)